server/catgenome/src/main/java/com/epam/catgenome/manager/blast/dto/BlastRequest.java

```diff
@@ -14,7 +14,7 @@
     private List<Long> taxIds;
     private List<Long> excludedTaxIds;
     private String query;
-    private Long maxTargetSequence;
-    private Long expectedThreshold;
+    private long maxTargetSequence;
```

Comment:
These values shall be objects  `Long` & `Double` not primitives, as `0` will be used as default value for primitive types and this will lead to request failure, as `0` is not a valid value for these parameters.